packages/protocol/remappings.txt

```diff
@@ -0,0 +1,6 @@
+@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable/
```

Comment:
how about "ozu" and "oz"?

Reply:
i would prefer `@openzepplin/(contracts|contracts-upgradeable)/*` because:
- readability > brevity for smart contract code
- this is the name of the npm package by openzeppelin